docs/reference-architectures/sap/index.md

```diff
@@ -0,0 +1,418 @@
+    own network in the cloud—that is, a logical isolation of the Azure cloud
+    dedicated to your subscription. You can also connect virtual networks to
+    your network on premises.
+
+-   **Subnets**. A subnet defines a logical collections of network resources by
+    specifying address ranges within a virtual network. Subnets make it easier
+    to manage and segregate network traffic without impacting performance. For
+    example, you can create a separate subnet for each tier: application (SAP
+    NetWeaver), database (SAP HANA), and management (the jumpbox).
+
+-   **Availability sets**. These are groups of virtual machines performing the
+    same role. Using availability sets improves the availability of a service
+    during system failure or routine cloud platform maintenance.
+
+-	**Load balancers.** The SAP Web Dispatcher component is used as a load balancer 
+    for SAP traffic among the SAP application servers. To achieve high availability,
+    the [Azure Load Balancer][azure-lb] (shown on the left in Figure 1) is used
+    to implement the failover cluster for the SAP web dispatchers. 
+    SMLG, the internal load balancer in ABAP
+    Central Services (ASCS), is used for balancing the SAP application server
+    pool for SAPGUIs and RFC traffic.  An Azure [internal load balancer][ilb]
+     (shown on the right in Figure 1) is also used to implement the ASCS Windows
+    Server Failover Cluster (WSFC). The application server connection to the
+    highly available ASCS is through the cluster virtual network name. Assigning
+    the cluster virtual network name to the endpoint of this internal load
+    balancer is optional.
+
+-	**NICs.** Network interface cards (NICs) enable all communication of virtual
+    machines on a virtual network.
+
+-   **Network security groups**. To restrict traffic between the various subnets
+    in the virtual network, you can create [network security
+    groups][nsg] (NSGs).
+
+    
+-	**VPN.** To extend on-premises networks into Azure over a dedicated private
+    connection, a VPN is created. See our [planning and design][planning] recommendations.
+
+    > [!NOTE]
+    > [ExpressRoute][expressroute] is the recommended Azure service for creating private connections that do not go over the public Internet.
+    > 
+   
+-   **Jumpbox**. In the management subnet, this secure virtual machine on the
+    network (also called a bastion host) allows administrators to connect to the
+    other virtual machines for administrative tasks only.
+
+-   **Azure virtual machines**. Windows Azure virtual machines are used for
+    administration and SAP workloads. The database runs on Linux Azure
+    virtual machines.
+
+-   **Azure Storage**. This is a persistent store. The operating system, SAP
+    application, and business data are stored in [Azure Storage][azure-storage].
+
+-   **SIOS DataKeeper**. To support the WSFC environment, SIOS DataKeeper
```

Comment:
Is DataKeeper a software component? I think this point belongs somewhere in recommendations, not here, as it's not in the diagram and not an infrastructure component.